src/main/java/uk/ac/ebi/embl/converter/gff3toff/GFF3Mapper.java

```diff
@@ -59,25 +61,24 @@ public Entry mapGFF3ToEntry(GFF3Annotation annotation) {
                     String accessionId = accession.substring(0, accession.lastIndexOf('.'));
                     entry.setPrimaryAccession(accessionId);
                     Location location = this.locationFactory.createLocalRange(reg.start(), reg.end());
-                    Order<Location> compoundJoin = new Order();
+                    Join<Location> compoundJoin = new Join<>();
                     compoundJoin.addLocation(location);
                     sourceFeature.setLocations(compoundJoin);
                 }
             }
             entry.addFeature(sourceFeature);
         }
 
-        for (GFF3Feature feature : annotation.getFeatures()) {
-            if (feature.getId().isPresent()) {
-                parentFeatures.put(feature.getId().get(), feature);
+        for (GFF3Feature gff3Feature : annotation.getFeatures()) {
+            if (gff3Feature.getId().isPresent()) {
+                parentFeatures.put(gff3Feature.getId().get(), gff3Feature);
             }
 
-            entry.addFeature(mapGFF3Feature(feature));
+            mapGFF3Feature(gff3Feature);
 
 
-            for (GFF3Feature childFeature : feature.getChildren()) {
-                Feature ffChildFeature = mapGFF3Feature(childFeature);
-                entry.addFeature(ffChildFeature);
+            for (GFF3Feature childFeature : gff3Feature.getChildren()) {
+                mapGFF3Feature(childFeature);
```

Comment:
These lines are not needed as we are adding all Features to gff3Annotation object. getChildren will me empty always

Reply:
In the GFF3 to FF conversion, we are not considering parent-child relationships at present, and I don't see any practical use for these lines. Could you please remove them? we can always add them later if needed.